neo4j/__init__.py

```diff
@@ -394,7 +394,7 @@ class Neo4jDriver(Routing, Driver):
     @classmethod
     def open(cls, *targets, auth=None, routing_context=None, **config):
         from neo4j.io import Neo4jPool
-        from neo4j.work import WorkspaceConfig
+        from neo4j import WorkspaceConfig
```

Comment:
Needed? Another import? WorkspaceConfig is imported on top but from neo4j.conf.

Reply:
Oh nice spotted

Reply:
it is resolved.